lib/core-integration/src/Test/Integration/Scenario/API/Byron/Migrations.hs

```diff
@@ -280,7 +275,7 @@ spec = describe "BYRON_MIGRATIONS" $ do
             payloadMigrate
         verify rm
             [ expectResponseCode HTTP.status202
-            , expectField id ( (`shouldBe` (numOfTxs)) . length )
+            , expectField id ( (`shouldBe` 19) . length )
```

Comment:
why not `20` ? :thinking:

Reply:
Because this PR changes the fee calculation a bit and it no longer generate 20 transactions but 19 :shrug:

Reply:
yes, I see it now